test/e2e/Makefile

```
@@ -0,0 +1,47 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# If you update this file, please follow:
+# https://suva.sh/posts/well-documented-makefiles/
+
+# Use GOPROXY environment variable if set
+
+.DEFAULT_GOAL:=help
+
+GOPROXY := $(shell go env GOPROXY)
+ifeq ($(GOPROXY),)
+GOPROXY := https://proxy.golang.org
+endif
+export GOPROXY
+
+REPO_ROOT := $(shell git rev-parse --show-toplevel)
+
+
+help:  ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+## --------------------------------------
+## Testing
+## --------------------------------------
+
+GINKGO_FOCUS  ?=
+E2E_CONF_FILE  ?= ${REPO_ROOT}/test/e2e/config/docker-dev.conf
+ARTIFACTS_FOLDER ?= ${REPO_ROOT}/_artifacts
+SKIP_RESOURCE_CLEANUP ?= false
+
+.PHONY: run-e2e
+run-e2e: ## Run the end-to-end tests
```

```suggestion
.PHONY: run
run: ## Run the end-to-end tests
```

given that we're already in the e2e folder

---

done